src/DataStructures/Tensor/Expressions/TensorExpression.hpp

```diff
@@ -93,10 +93,14 @@ struct TensorIndex {
  */
 SPECTRE_ALWAYS_INLINE static constexpr size_t
 get_tensorindex_value_with_opposite_valence(const size_t& i) noexcept {
```

Comment:
when you squash: you don't need to take types like `size_t` and `double` by reference because references are effectively pointers, pointers are 8 bytes in size, just like `size_t` and `double`, so it's actually more efficient to "copy" the value than to create a pointer. In practice, since this is always inlined it probably won't matter, but a useful FYI :)

Reply:
Good point regarding the equivalent number of bytes. I'll update it regardless of the inlining